api/posts.go

```diff
@@ -7,13 +7,16 @@ import (
 	"fmt"
 	"io"
 	"log"
+	"time"
 
 	"gocloud.dev/blob"
 	"gocloud.dev/pubsub"
 
 	"github.com/ourrootsorg/cms-server/model"
 )
 
+const ImagesPrefix = "/images/%d/"
```

Comment:
I thought you had decided to put images under the posts, so /posts/id/images/filename?

Reply:
Totally fine where they are as well; I'm not asking for a change here.

Reply:
`ImagesPrefix` is the S3 prefix, so the S3 files are named `/images/<postID>/<path in ZIP file>`. The URL to retrieve them is `/posts/<postID>/images/<path in ZIP file>`. I can make the S3 path match the URL if you think that's valuable.

Reply:
No, I don't think so. That looks good